e2e/pierced-react-netlify/README.md

```diff
@@ -0,0 +1,6 @@
+# pierced-react-netlify app
```

Comment:
This is a bit surprising - was it really necessary to fork all the examples and create a "netlify" version of them?

If so, why? I'd expect all examples to be hosting platform agnostic but your change implies that that's not the case. What could we change in the examples to make them netlify compatible without this duplication?

Reply:
Fwiw, I didn’t really have to do anything other than run the framework specific generators for Netlify. Which pulls in the right netlify adapters and such for each. But again, it was more about getting past this part to focus on the web fragments bit